omod/src/main/webapp/pages/forms/extension.gsp

```diff
@@ -13,23 +13,23 @@
 <h2>${ui.encodeHtmlContent(form.name)} - ${ ui.message("formentryapp.addform") }</h2>
 
 <form method="post" action="${ ui.pageLink("formentryapp", "forms/extension") }">
-<input type="hidden" name="extensionForm.form" value="${form.id}" />
-<input type="hidden" name="extensionForm.id" value="${extensionForm.id}" />
+<input type="hidden" name="extensionForm.form" value="${ui.escapeAttribute(form.id)}" />
+<input type="hidden" name="extensionForm.id" value="${ui.escapeAttribute(extensionForm.id)}" />
 
 <p>
 <label name="extensionForm.formTechnology">${ui.message("formentryapp.formtechnology")}: ${formTechnology}<label>
 </p>
 
 <p>
-<label name="extensionForm.uiLocation">${ui.message("formentryapp.uilocation")}: 
+<label name="extensionForm.uiLocation">${ui.message("formentryapp.uilocation")}:
 <% if (extensionForm.id) { %>
 ${ui.message("formentryapp." + extensionForm.uiLocation)}
-<input type="hidden" id="extensionForm.uiLocation" name="extensionForm.uiLocation" value="${extensionForm.uiLocation}" />
+<input type="hidden" id="extensionForm.uiLocation" name="extensionForm.uiLocation" value="${ui.escapeAttribute(extensionForm.uiLocation)}" />
```

Comment:
Is this also necessary for the hidden input type?

Reply:
I'd still say yes to this. it's definitely necessary to escape quotes for the same reason above. specifics of the tag shouldn't matter.